source/UIDataCodeGen/CodeGen/InstantiatorGeneratorBase.cs

```diff
@@ -373,6 +375,9 @@ protected string GenerateCode()
             // Write the LoadedImageSurface byte arrays into the outer (IAnimatedVisualSource) class.
             WriteLoadedImageSurfaceArrays(builder);
 
+            // Write the method that binds AnimationController.Progress of animations to an expression.
```

Comment:
I would change the wording here a bit. How about:
```suggestion
            // Write the method that binds the AnimationController.Progress property of a given animation to an expression.
```

Reply:
It starts the animation as well.

---
In reply to: [369875297](https://github.com/windows-toolkit/Lottie-Windows/pull/224#discussion_r369875297) [](ancestors = 369875297)